mkdocs_lumi/home.html

```diff
@@ -72,7 +72,7 @@
       <p align="center">
         <!-- You have landed on the home page of the documentation of the 3<sup>rd</sup> fastest supercomputer in the World! <br /> -->
         
-        These pages contain instructions, user guides and tutorials for the use of the LUMI supercomputer.
+        You have reached the user guides and tutorials for the LUMI supercomputer.
```

Comment:
Suggestions to think of the text on the home page:

Here you can find user guides and tutorials for the LUMI supercomputer.
These pages contain user guides and tutorials for the LUMI supercomputer.
You have found your way to the user guides and tutorials for the LUMI supercomputer. 
From these pages you can find user guides and tutorials for the use of LUMI supercomputer. 
On these pages you can find help and instructions for using the LUMI supercomputer.

Reply:
I like `You have found your way to the user guides and tutorials for the LUMI supercomputer.` I will use that one.